### ISSUE TITLE
Network and port prereqs are missing

### ISSUE THREAD

**Issue by Yupiter2005** [MicrosoftDocs/SCCMdocs#1385]
As well as a good guide which buttons should be clicked in DP properties

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 15426bd1-cbdb-3661-375b-886be88a8c9a
* Version Independent ID: 2fed8449-b628-0fd2-4057-4a94262ec662
* Content: [Use multicast to deploy Windows over the network - Configuration Manager](https://docs.microsoft.com/en-us/sccm/osd/deploy-use/use-multicast-to-deploy-windows-over-the-network)
* Content Source: [sccm/osd/deploy-use/use-multicast-to-deploy-windows-over-the-network.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/osd/deploy-use/use-multicast-to-deploy-windows-over-the-network.md)
* Product: **configuration-manager**
* Technology: **configmgr-osd**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @Yupiter2005 for the feedback. This specific article is intended to be more conceptual. In the section to **Configure a distribution point to support multicast**, there's a link to an article with the specific details of which knobs to use in the DP properties. The problem that I see here, is that you have to go through an intermediate page before you get to the real details in the [Multicast](https://docs.microsoft.com/sccm/core/servers/deploy/configure/install-and-configure-distribution-points#bkmk_config-multicast) section of the Install and configure DPs article. 

From a planning perspective, you make a good point that this article should note the ports that are required, which are listed in the [Ports](https://docs.microsoft.com/sccm/core/plan-design/hierarchy/ports#BKMK_PortsClient-DP2) reference article, so a link over to that would be useful. 

So basically I'm interpreting this feedback as changing one link, and adding another link.

**Comment by aczechowski** [comment]
Thanks to @jpcast88 for making this change, which should publish in less than an hour
